gcc/testsuite/rust/compile/unicode_norm1.rs

```diff
@@ -0,0 +1,6 @@
+fn main() {
+    // U+304C
+    let が = ();
+    // U+304B + U+3099
+    let _ = が;
+}
```

Comment:
This code compiles despite that these two identifiers has different byte string.
It means identifier normalization seems to work.